GenderPayGap.UnitTests/GenderPayGap.WebUI.Tests/Builders/UserOrganisationBuilder.cs

```diff
@@ -0,0 +1,50 @@
+﻿using System;
+using GenderPayGap.Database;
+
+namespace GenderPayGap.WebUI.Tests.Builders
+{
+    public class UserOrganisationBuilder
+    {
+
+        private long userId = 1;
+        private User user = new UserBuilder().Build();

```

Comment:
It would be nice if these two properties were more linked.

e.g. you could remove `userId` and, in the `Build` method, say
```csharp
return new UserOrganisation
{
	UserId = user.UserId, 
	User = user,
	...
};
```

And the same for `organisation` / `organisationId`